homeassistant/components/deconz/diagnostics.py

```diff
@@ -25,7 +25,8 @@ async def async_get_config_entry_diagnostics(
     diag["deconz_config"] = async_redact_data(
         gateway.api.config.raw, REDACT_DECONZ_CONFIG
     )
-    diag["websocket_state"] = gateway.api.websocket.state
+    if gateway.api.websocket:
```

Comment:
Is it worth setting `diag["websocket_state"]` to some sentinel value (like "UNKNOWN") for the case when `gateway.api.websocket` is invalid?

Reply:
This is one aspect of the library which hasn't been rewrite. Yet in order to keep the breaking changes smaller. I intend to move this around so the state value should always be available

Reply:
Simplified this in the mean time, I agree its better to always publish the values